solutions/swb-reference/README.md

```diff
@@ -6,67 +6,566 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
+LaunchConstraintRolePrefix: *
+StatusHandlerRoleArn: <CFN_OUTPUT.StatusHandlerLambdaRoleOutput>
+```
+
+After the deployment is complete, take note of the following Cloudformation outputs. The outputs are on the `Outputs` tab. We will be using in future steps
+```
+HostingAccountHandlerRoleArn
+EncryptionKeyArn
+EnvMgmtRoleArn
+VPC
+VpcSubnet
+```
+### Configuring App Registry applications limits
+
+Service Workbench v2.0 on AWS uses [AWS App Registry](https://docs.aws.amazon.com/servicecatalog/latest/arguide/intro-app-registry.html) applications to group and add metadata and attributes to created resources.
+By using App Registry, Service Workbench is able to organize its resources and track their dependencies more efficiently.
+Every resource created in Service Workbench is associated to an App Registry application including all workspaces.
+App Registry currently has a default limit of 1000 resources per application.
+
+If you are estimating to have more than 999 Workspaces created in your Service Workbench instance a service quota increase will be needed.
+Follow these steps to request a quota increase for App Registry application resources:
+
+1. Sign in to your **Hosting Account** in the [AWS Management Console](https://console.aws.amazon.com/console/home?nc2=h_ct&src=header-signin).
+
+1. Open the [**Service Quotas console**](https://console.aws.amazon.com/servicequotas/home).
+
+1. In the search, enter `AWS Service Catalog` and choose **Service Catalog** from the results.
+
+1. Under **Services Quota**, choose **Resources per application**.
+
+1. On the **Resources per application** page, choose **Request Quota Increase** under **Recent quota increase requests**.
+
+1. In **Change quota**, enter your estimated number of workspaces plus 1 (infrastructure resource).
+
+1. Choose **Request**.
 
-1. In `solution-spark-on-aws` root directory run `rush build`
-2. In `solution-spark-on-aws/solutions/swb-reference` root directory run `STAGE=<STAGE TO RUN LOCALLY> ./scripts/runLocally.sh`. This will run a local lambda server.
 
-## Deploying Code
 
-Run one time to Bootstrap the CDK
+## Get access token
 
-`STAGE=<STAGE> rushx cdk bootstrap`
+To get the `accessToken`, `csrfCookie`, and `csrfToken` for making authenticated API requests please refer [here](README.md#obtain-access-token-for-making-authenticated-api-requests).  
+
+## POSTMAN Setup
+In POSTMAN create an environment using the instructions [here](https://learning.postman.com/docs/sending-requests/managing-environments/#creating-environments).
+Your environment should have four variables. Name the first one `API_URL` and the value should be the `APIGatewayAPIEndpoint` value that you got when deploying the `Main Account`. Name the second, third and fourth ones `ACCESS_TOKEN`, `CSRF_COOKIE`, and `CSRF_TOKEN` and their values should be the `accessToken`, `csrfCookie`, and `csrfToken` you got from [Setup UI and Get Access Token](#setup-ui-and-get-access-token)
```

Comment:
not sure if it's just me but i need to remove the ending '/' from APIGatewayAPIEndpoint (https://xxx.execute-api.<region>.amazonaws.com/dev/) to get API_URL (https://xxx.execute-api.<region>.amazonaws.com/dev)

Reply:
May be a version of your Postman, I have the `/` at the end and my Postman collection is working